app/src/main/java/com/epfl/drawyourpath/database/FireDatabase.kt

```diff
@@ -23,6 +34,10 @@ class FireDatabase : Database() {
         return future
     }
 
+    override fun updateUsername(username: String, userId: String): CompletableFuture<Boolean> {
+        TODO("This will be implmented during next task when I will clean the database organisation")
+    }
+
     override fun setUserName(userName: String) {
```

Comment:
Inconsistency in casing. Should be `setUsername()` to match other function names